configure.ac

```diff
@@ -1048,6 +1048,36 @@ AM_CONDITIONAL([OPENTHREAD_ENABLE_RAW_LINK_API], [test "${enable_raw_link_api}"
 AC_DEFINE_UNQUOTED([OPENTHREAD_ENABLE_RAW_LINK_API],[${OPENTHREAD_ENABLE_RAW_LINK_API}],[Define to 1 if you want to enable raw link-layer API])
 
 #
+# Passing up ACK frame (This option should be removed after all platforms could pass up ACK frame)
+#
+
+AC_ARG_ENABLE(passing_up_ack,
+    [AS_HELP_STRING([--enable-passing-up-ack],[Enable passing up ack support @<:@default=yes@:>@.])],
```

Comment:
This config is enabled by default, only the platforms which do not pass up ACK frame will need to add `--enable-passing-up-ack=no` build option.